hydra-redis/test/querying_test.py

```diff
@@ -0,0 +1,127 @@
+import unittest
+import urllib.request
+import json
+from hydrus.hydraspec import doc_maker
+from os import sys, path
+
+PARENT_DIR = path.dirname(path.dirname(path.abspath(__file__)))
+sys.path.append(PARENT_DIR)
+import querying_mechanism
```

Comment:
Use absolute imports everywhere not relative imports

Reply:
I am trouble here, I have tried or read many things but I don't find any for that. Please help for that.